app/views/foreman_leapp/job_templates/upgrade.erb

```diff
@@ -9,8 +9,9 @@ template_inputs:
 - name: reboot
   description: reboot the vm automaticaly to continue with the upgrade
```

Comment:
Could you please change this to "Reboot the host automaticaly to continue with the upgrade", it's not always VM, also please capital R.